nsv13/code/modules/research/techweb/all_nsv_nodes.dm

```diff
@@ -154,3 +154,48 @@
 	prereq_ids = list("fightergun2", "adv_weaponry")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
 	export_price = 5000
+
+//Vehicle Hardpoint tech
+/datum/techweb_node/vehicle_start
+	id = "vehicle_start"
+	display_name = "Mechanical Research"
+	description = "Researching the possibility of expanding the usage behind those useless Tugs"
+	prereq_ids = list("engineering")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1500)
+	export_price = 1000
```

Comment:
This should unlock the 'pathetic' tug engine IMO.

Reply:
Altered it so this one unlocks the pathetic tug engine and the utilities don't